modules/operations/file_ops.rb

```diff
@@ -0,0 +1,11 @@
+require 'json'
+
+def open_file(path)
+  if File.exist?(path)
+    json_string = File.read(path)
+    json_string != '' && json_string != 'null' ? JSON.parse(json_string) : []
+  else
+    File.new(path, 'w')
+    []
+  end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_preserve_data.md#:~:text=On%20exit%20from%20your%20app%20save%20all%20the%20data%20(books%2C%20people%2C%20and%20rentals)%20into%20JSON%20files), Please save the people, books, and rentals data on app exit, not just after creating it as you did.
[Hint] In order to do that you should add a method that saves those data and use it when a user exits the app.